lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
Oh. Does setting an environment variable work? Don't we need to add something around here https://github.com/stitchfix/asset_sync/blob/master/lib/asset_sync/engine.rb#L15-L43 to make that happen?

As far as where we call `rake assets:sync` goes, two options come to mind.

First, we can edit `circle.yml` as you suggest. I think it would make sense to perform the asset sync before we push to Heroku, so perhaps something like

```yaml
# circle.yml
    commands:
      - git fetch --unshallow
      - bundle exec rake assets:precompile
      - bundle exec rake assets:sync
      - git push git@heroku.com:fashionthing-production.git $CIRCLE_SHA1:refs/heads/master
```

Option 2 is we let Heroku handle the asset sync, but that means extending the existing `assets:precompile` task, something closer to what the author of Asset Sync PR 341 was trying to do. Heroku runs `assets:precompile` automatically when it fails to detect a manifest.json file in the compiled slug. [Details here.](https://devcenter.heroku.com/articles/rails-asset-pipeline#compiling-assets-locally)

If we pursue option 1, we might want to consider committing a dummy manifest.json to git in order to trick Heroku into not running that task. It will no longer be necessary.